enforcer-rules/pom.xml

```diff
@@ -103,9 +103,11 @@
       <groupId>org.apache.maven.resolver</groupId>
       <artifactId>maven-resolver-util</artifactId>
     </dependency>
+    <!-- needed for ArtifactCollector and maven-dependency-tree 2.2 -->
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-compat</artifactId>
+      <scope>compile</scope>
```

Comment:
`scope` doesn't seem to be managed to anything else in parent, so why is it needed here and not for `maven-resolver-util` above?